.github/workflows/verify_docker_image.yml

```
@@ -1,33 +1,114 @@
 name: Pull and Verify Docker Image
 
 on:
-  workflow_call:
   workflow_dispatch:
+    inputs:
+      amdgpu_target:
+        description: "AMDGPU target to verify"
+        required: true
+        default: "gfx942"
+        type: string
+      image_digest:
+        description: "Digest of image to verify"
+        required: true
+        type: string
+      runner_type:
+        description: "Runner type to use"
+        required: true
+        default: "linux-mi300-1gpu-ossci-rocm"
+        type: string
+      dispatch_id:
+        description: "Auto Generated Id for a dispatch"
+        type: string
+        required: true
+      version:
+        description: "Version Number as X.Y.ZrcYYYYMMDD format"
```

```suggestion
        description: "Version number string"
```
Would this work for dev releases which have a `X.Y.Z.dev0+${hash}` version number? If yes, we should adjust the description as above.